app/talk/discussion-preview.cjsx

```diff
@@ -62,8 +62,9 @@ module.exports = React.createClass
       <div className="preview-content">
 
         {if @state.subject?
+          subject = getSubjectLocation(@state.subject)
           <div className="subject-preview">
-            <Thumbnail src={getSubjectLocation(@state.subject).src} width={100} />
+            <Thumbnail src={subject.src} format={subject.format} width={100} controls={false} />
```

Comment:
The controls cover the video too much, and since this is a small Thumbnail, I figure it's important to show as much of the subject as possible. The Thumbnail can still be clicked to play.
![screen shot 2017-01-06 at 9 40 06 am](https://cloud.githubusercontent.com/assets/14099077/21722962/5a0d458e-d3f4-11e6-94d0-a79a26316349.png)